cdap-app-templates/cdap-program-report/cdap-program-report-base/src/main/scala/co/cask/cdap/report/ReportGenerationHelper.scala

```diff
@@ -130,9 +130,11 @@ object ReportGenerationHelper {
     // Writing the DataFrame to JSON files requires a non-existing directory to write report files.
     // Create a non-existing directory location with name ReportSparkHandler.REPORT_DIR
     val reportDir = reportIdDir.append(Constants.LocationName.REPORT_DIR).toURI.toString
-    // TODO: [CDAP-13290] output reports as avro instead of json text files
     // TODO: [CDAP-13291] improve how the number of partitions is configured
-    resultDf.coalesce(1).write.option("timestampFormat", "yyyy/MM/dd HH:mm:ss ZZ").json(reportDir)
+    // by default spart sql uses snappy compression codec, set it to uncompressed
+    sql.setConf("spark.sql.avro.compression.codec", "uncompressed")
+    resultDf.coalesce(1).write.option("timestampFormat",
+      "yyyy/MM/dd HH:mm:ss ZZ").format("com.databricks.spark.avro").save(reportDir)
```

Comment:
we're using databricks avro? Is this available in all distros? Or is this getting picked up from the app jar?

Reply:
i notice that `com.databricks` `spark-avro_2.11` is added as a dependency in application's pom.xml. so its coming from the app.